conda/recipes/rmm/meta.yaml

```
@@ -40,22 +40,36 @@ requirements:
     - ninja
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }} {{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} {{ cuda_version }}
+    {% else %}
+    - cuda-version {{ cuda_version }}
+    - {{ compiler('cuda') }}
+    {% endif %}
+
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
-    - cuda-python >=11.7.1,<12.0
+    {% if cuda_major == "11" %}
     - cudatoolkit ={{ cuda_version }}
+    - cuda-python >=11.7.1,<12.0
+    {% else %}
+    - cuda-python >=12.1,<13.0
+    {% endif %}
```

Since `cuda_python` is in the `conda_build_config.yaml` file an the proper version is aligned with the proper CUDA version, think we can simplify this logic

```suggestion
    {% endif %}
    - cuda-python
```